packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -71,12 +71,12 @@ type ChatMessageComponentAsMessageBubbleProps = {
    * @beta
    */
   onDisplayDateTimeString?: (messageDate: Date) => string;
-  /* @conditional-compile-remove(at-mention) */
+  /* @conditional-compile-remove(mention) */
   /**
-   * Optional props needed to display suggestions in the at mention scenario.
+   * Optional props needed to display suggestions in the mention scenario.
    * @beta
```

Comment:
Correct me if I am wrong, is this ChatMessageComponentAsMessageBubbleProps a beta api or not?
If not, we should mark it as @private or @internal for this props
Same as the props above "onDisplayDateTimeString" (although it's unrelated to this PR)